internal/diff.ts

```diff
@@ -147,22 +152,17 @@ function createFp(
  */
 export function diff<T>(A: T[], B: T[]): DiffResult<T>[] {
   const prefixCommon = createCommon(A, B);
-  const suffixCommon = createCommon(
```

Comment:
apparently this became unreachable with the most recent refactoring in https://github.com/denoland/deno_std/pull/4704 due to the lack of `reverse` logic.